src/auto-evo/RunResults.cs

```diff
@@ -182,6 +182,14 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                     builder.Append(previousPopulations.GetPatch(patch.ID).GetSpeciesPopulation(species));
                 }
 
+                if (population <= 0)
+                {
+                    builder.Append("\n");
+                    builder.Append("  ");
+                    builder.Append(" went extinct in ");
+                    builder.Append(PatchString(patch));
```

Comment:
I think that when this is printed, the population number should be left out.
So this should be done something like:

```
              if (population > 0){
                builder.Append(" population: ");
                builder.Append(Math.Max(population, 0));
} else {
                    builder.Append("\n");
                    builder.Append("  ");
                    builder.Append(" went extinct in ");
}
                if (previousPopulations != null)
                {
                    builder.Append(" previous: ");
                    builder.Append(previousPopulations.GetPatch(patch.ID).GetSpeciesPopulation(species));
                }
```